plugins/obs-qsv11/CMakeLists.txt

```diff
@@ -2,12 +2,24 @@ cmake_minimum_required(VERSION 3.22...3.25)
 
 legacy_check()
 
-option(ENABLE_QSV11 "Build Intel QSV11 Hardware Encoder." TRUE)
+if(NOT OS_LINUX OR CMAKE_SYSTEM_PROCESSOR MATCHES "^(i[3-6]86|x86|x64|x86_64|amd64|AMD64)$")
```

Comment:
I actually thought this through another time and I think the better way would be to do handle this in the main plugins CMakeLists file with something similar to `check_obs_browser`:

* The plugin modules themselves should be almost agnostic as much as possible
* The decision whether to load a plugin for a specific OS or architecture should be made in the place that "selects" which plugin directories to include.

Reply:
I wait for the aarch64 CI on my fork to finish to check the theoretical cache size. So I will not push right now.

Here is what I done for the macro in the plugins CMakeLists, what do you think ?
```cmake
macro(check_obs_qsv11)
    if(OS_WINDOWS
       OR OS_LINUX
       AND CMAKE_SYSTEM_PROCESSOR MATCHES "^(i[3-6]86|x86|x64|x86_64|amd64|AMD64)$")
      add_subdirectory(obs-qsv11)
    endif()
  endmacro()
```

Reply:
The change was pushed.